build-info-extractor/src/test/java/org/jfrog/build/extractor/clientConfiguration/util/UrlUtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package org.jfrog.build.extractor.clientConfiguration.util;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Created by Bar Belity on 19/07/2020.
+ */
+public class UrlUtilsTest {
```

Comment:
UrlUtilsTest --> UrlUtils

Reply:
Decided to leave test classes with 'Test' suffix.